### ISSUE TITLE
Popular shared/common template paths (e.g. app/views/shared, etc.)

### ISSUE THREAD

**Issue by eliotsykes** [eliotsykes/real-world-rails#36]
What directory paths are popular for storing shared templates?

**Comment by eliotsykes** [comment]
I should write an inspector, but just trying this out by hand with `ls -alR | grep 'app/views'`:

(Majority, if not all apps using devise will be making use of `app/views/devise/shared`)

```
./24pullrequests/app/views/shared
./amahi-platform/app/views/shared
./asciinema.org/app/views/shared
./askthem/app/views/shared
./bridge_troll/app/views/shared
./canvas-lms/app/views/shared:
./canvas-lms/app/views/shared/errors:
./canvas-lms/app/views/shared/svg:
./canvas-lms/app/views/shared/svg/k12
# sorry, time to get on with other work
```
```
# Candidates, need to check
./alaveteli/app/views/general
./alaveteli/app/views/widgets
./amahi-platform/app/views/js_templates/shares
```

Investigate that each of the shared dirs contain files and what those files typically do. Any repeating files/purposes, e.g. nav templates?

**Comment by eliotsykes** [comment]
Closed by #38
